misc/tutorial/401_AllFeatures.ngdoc

```diff
@@ -15,44 +15,52 @@ All features are enabled to get an idea of performance
 
     app.controller('MainCtrl',  ['$scope', '$http', '$timeout', '$interval', 'uiGridConstants', 'uiGridGroupingConstants',
      function ($scope, $http, $timeout, $interval, uiGridConstants, uiGridGroupingConstants) {
-
-      $scope.gridOptions = {};
-      $scope.gridOptions.data = 'myData';
-      $scope.gridOptions.enableCellEditOnFocus = true;
-      $scope.gridOptions.enableColumnResizing = true;
-      $scope.gridOptions.enableFiltering = true;
-      $scope.gridOptions.enableGridMenu = true;
-      $scope.gridOptions.showGridFooter = true;
-      $scope.gridOptions.showColumnFooter = true;
-      $scope.gridOptions.fastWatch = true;
-
-      $scope.gridOptions.rowIdentity = function(row) {
-        return row.id;
+       var gridApi;
+
+      $scope.gridOptions = {
```

Comment:
I decided to combine the multiple calls to $scope.gridOptions.[something] into one object definition.